eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver/SiteAccessGroupConfigResolver.php

```diff
@@ -21,6 +22,58 @@ class SiteAccessGroupConfigResolver extends SiteAccessConfigResolver
 {
     use ContainerAwareTrait;
 
+    /** @var string[][] */
+    protected $siteAccessGroups;
+
+    public function __construct(
+        SiteAccess\SiteAccessProviderInterface $siteAccessProvider,
+        string $defaultNamespace,
+        array $siteAccessGroups
+    ) {
+        parent::__construct($siteAccessProvider, $defaultNamespace);
+        $this->siteAccessGroups = $siteAccessGroups;
+    }
+
+    final public function hasParameter(string $paramName, ?string $namespace = null, ?string $scope = null): bool
+    {
+        [$namespace, $scope] = $this->resolveNamespaceAndScope($namespace, $scope);
+
+        if ($this->isSiteAccessGroupScope($scope)) {
+            return $this->resolverHasParameterForGroup(new SiteAccessGroup($scope), $paramName, $namespace);
+        }
+
+        if (!$this->isSiteAccessScope($scope)) {
+            return false;
+        }
+
+        $siteAccess = $this->siteAccessProvider->getSiteAccess($scope);
+        if (!$this->isSiteAccessSupported($siteAccess)) {
+            return false;
+        }
+
+        return $this->resolverHasParameter($siteAccess, $paramName, $namespace);
```

Comment:
This seems redundant, why not:
```suggestion
        return parent::hasParameter($paramName, $namespace, $scope);
```